core/domain/exp_services_test.py

```diff
@@ -1673,6 +1682,17 @@ class ExplorationCommitLogUnitTests(ExplorationServicesUnitTests):
         'post_commit_status': 'public'
     }
 
+    COMMIT_UPDATE_FIRST_PUBLISHED_MSEC_EXP_2 = {
```

Comment:
What you said in your last comment sounds good. (Also, thanks for the PR updates -- I'll take a closer look tonight!)